### PR TITLE
Making test table driven

### PR DESCRIPTION
**Pull request by reetasingh**
making test table drive and adding more test cases

### CODE REVIEW

**Comment by aveshagarwal** [comment]
thanks, will review it soon,.

File: pkg/descheduler/strategies/pod_antiaffinity_test.go [hunk 1]
```diff
@@ -44,26 +44,54 @@ func TestPodAntiAffinity(t *testing.T) {
...
+		},
 	}
-	npe[node] = 0
-	expectedEvictedPodCount = 1
-	podsEvicted = removePodsWithAffinityRules(fakeClient, "v1", []*v1.Node{node}, false, npe, 1, false)
-	if podsEvicted != expectedEvictedPodCount {
-		t.Errorf("Unexpected no of pods evicted: pods evicted: %d, expected: %d", podsEvicted, expectedEvictedPodCount)
+
+	for _, test := range tests {
+
```
Comment (damemi) [inline pkg/descheduler/strategies/pod_antiaffinity_test.go]: extra line?

**Comment by seanmalloy** [comment]
/remove-lifecycle rotten

@reetasingh are you planning to continue work on this pull request? Would you be able to rebase and fix the merge conflicts?

Thanks!

**Comment by ingvagabund** [comment]
@reetasingh thanks for pointing the missing table driven tests implementation. Portion of it was implemented through https://github.com/kubernetes-sigs/descheduler/pull/266. Please, feel free to extend the list of cases in case I missed some.

/close

**Pull request closed without merging**